File: turisticka_agencija_mobile/lib/models/Komentar.dart
import 'package:json_annotation/json_annotation.dart';

part 'Komentar.g.dart';

@JsonSerializable()
class Komentar {
  String datum;
  int? id;
  int korisnikId;
  int putovanjeId;
  String sadrzaj;

  Komentar(
      {required this.datum,
      this.id,
      required this.korisnikId,
      required this.putovanjeId,
      required this.sadrzaj});

  factory Komentar.fromJson(Map<String, dynamic> json) => _$KomentarFromJson(json);

  Map<String, dynamic> toJson() => _$KomentarToJson(this);
}

File: TuristickaAgencija.Model/Komentar.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
   public class Komentar
    {
        public int Id { get; set; }
        public int PutovanjeId { get; set; }
        public int KorisnikId { get; set; }
        public DateTime Datum { get; set; }
        public string Sadrzaj { get; set; }
    }
}
